packages/schematics/migrations/update-8-3-0/update-8-3-0.ts

```diff
@@ -0,0 +1,31 @@
+import { Rule, SchematicContext, Tree } from '@angular-devkit/schematics';
+
+function updateJestConfig(host: Tree, context: SchematicContext) {
```

Comment:
Can you move this migration to `@nrwl/jest` please?

Reply:
Done with latest commit